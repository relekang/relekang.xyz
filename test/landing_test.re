open Jest;

open ExpectJs;

describe
  "<Landing />"
  (
    fun _ =>
      test
        "render matches snapshot"
        (
          fun _ => {
            let component = ReactShallowRenderer.renderWithRenderer <Landing />;
            expect component |> toMatchSnapshot
          }
        )
  );
