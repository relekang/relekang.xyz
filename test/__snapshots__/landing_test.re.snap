// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> render matches snapshot 1`] = `
<div>
  hi there
</div>
`;
